kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/model/RegisteredProperty.kt

```diff
@@ -0,0 +1,13 @@
+package godot.entrygenerator.model
+
+data class RegisteredProperty(
+    val fqName: String,
+    val type: Type,
+    val isMutable: Boolean = true,
```

Comment:
Nice we are even prepared to the case Godot add read-only properties
(Sadly they are against the keyword "val" as it's too close to "var" :( )

Reply:
Yeah but the main reason was that i need that information for the export check: https://github.com/utopia-rise/godot-kotlin-jvm/blob/ed18f34f64316d8e0c022eccca551531dff3f043/kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/checks/ExportedMutablilityCheck.kt#L14